core/network/fabric-interop-cc/contracts/interop/membership.go

```diff
@@ -96,8 +117,12 @@ func (s *SmartContract) GetMembershipBySecurityDomain(ctx contractapi.Transactio
 
 // verifyMemberInSecurityDomain function verifies the identity of the requester according to
 // the Membership for the external network the request originated from.
-func verifyMemberInSecurityDomain(s *SmartContract, ctx contractapi.TransactionContextInterface, cert *x509.Certificate, securityDomain string, requestingOrg string) error {
-	err := isCertificateWithinExpiry(cert)
+func verifyMemberInSecurityDomain(s *SmartContract, ctx contractapi.TransactionContextInterface, certPEM string, securityDomain string, requestingOrg string) error {
+	cert, err := parseCert(certPEM)
```

Comment:
Why do you want to pass string and parse cert here, given that all places that this function is being used they have already parsed the cert for some other uses. Would be redundant to parse it here also.

Reply:
The only reason for this change is to run the comparison in line 143. I agree with you about the redundancy otherwise. Let me know if there's a better way of doing things.